File: Month-1/Week-1/day-02/php/App/FindNemo.php
<?php
function FindNemo($text = ""){
    $nemo = "Nemo";
    $explodeText = explode(" ", $text);

    foreach($explodeText as  $key=>$value){
        if($value == $nemo){
            $position = $key + 1;
            return "I found Nemo at ".$position;
        }
    }
    
    return "I can't find Nemo :(";
}

File: Month-1/Week-1/day-02/haskell/FindNemo.hs
module FindNemo
  ( findNemo
  ) where

import           Data.List

findNemo :: String -> String
findNemo sentence = case elemIndex "Nemo" (words sentence) of
  Nothing    -> "I can't find Nemo :("
  Just index -> "I found Nemo at " ++ show (succ index) ++ "!"
